Gopkg.toml

```diff
@@ -245,7 +249,7 @@ required = ["github.com/centrifuge/centrifuge-ethereum-contracts", "github.com/r
 
 [[constraint]]
   name = "golang.org/x/crypto"
-  revision = "0e37d006457bf46f9e6692014ba72ef82c33022c"
+  branch = "master"
```

Comment:
I rather lock a version, but since go-substrate-rpc-client uses master, I can't do in this PR. I will create another one agains the GSRPC repo to lock that crypto version

Reply:
https://github.com/centrifuge/go-substrate-rpc-client/pull/34

Reply:
You can replace `constraint` with `override`
That will lock the version we give here